__init__.py

```diff
@@ -92,261 +93,263 @@ def user_request_handling(self, message):
+        variable contains the name of the store.
+        The matching function get_store_data() is
+        used to find the store name in cache or
         on the mall page.
         If user's request is not None this function
-        can return several shops, one shop or empty
+        can return several stores, one store or empty
         list.
-        If no shop was found asks user to repeat.
+        If no store was found asks user to repeat.
         returns 1, user_request to continue the
         execution loop in self.execute().
-        If there are several shops asks user what way
+        If there are several stores asks user what way
         of sorting to choose: time, level, nothing.
-            If 'time' - finds open shops. If open shops
-            list is not empty speaks open shops, else
-            tells time difference between user and shops'
+            If 'time' - finds open stores. If open stores
+            list is not empty speaks open stores, else
+            tells time difference between user and stores'
             work hours.
             If 'location' - asks what level user is interested
-            in. If shops were found speaks shops' info,
-            else tells that there is no shop on that level
-            and speaks all found shops.
+            in. If stores were found speaks stores' info,
+            else tells that there is no store on that level
+            and speaks all found stores.
             If  'no' - sorts by time.
             If nothing matched in the answer - sorts by time.
-        If there was one shop found speaks this
-        shop info. Returns 3, None to stop current
-        shop search.
+        If there was one store found speaks this
+        store info. Returns 3, None to stop current
+        store search.
         Location and time sorting functions return
-        3, None to stop current shop search.
+        3, None to stop current store search.
         """
         LOG.info(f'user_request {user_request}')
         LOG.info(f'mall_link {mall_link}')
         if user_request is not None:
             self.speak_dialog("start_parsing")
-            LOG.info(f"I am parsing shops and malls for your request")
+            LOG.info(f"I am parsing stores and malls for your request")
             file_path = self.file_system.path
             LOG.info(f'file_path {file_path}')
-            shop_info = get_shop_data(mall_link, user_request, file_path)
-            LOG.info(f"I found {len(shop_info)} shops")
-            LOG.info(f"shop list: {shop_info}")
-            if len(shop_info) == 0:
-                user_request = self.get_response('shop_not_found')
+            store_info = get_store_data(mall_link, user_request, file_path)
+            LOG.info(f"I found {len(store_info)} stores")
+            LOG.info(f"store list: {store_info}")
+            if len(store_info) == 0:
+                user_request = self.get_response('store_not_found')
```

Comment:
Dialog changed, so this needs to include:
```python
data={'store_name': store_name}
````

I'm not sure if we know what store the user is requesting in this case though, so perhaps the answer is to update the dialog file?